ontology.ttl

```diff
@@ -77,42 +102,148 @@
+  rdfs:subClassOf :Role .
+
+:ProjectMember a rdfs:Class ;
+  rdfs:comment "A contribution from a contributor on membership list of a designated project/project team involved in the creation of a resource" ;
+  rdfs:subClassOf :Role .
+
+# Not sure about this; Registration Agency is the problematic concept
+:RegistrationAgency a rdfs:Class ;
+  rdfs:comment "Institution/organisation officially appointed by a Registration Authority to handle specific tasks within a defined area of responsibility" ;
+  rdfs:subClassOf :Role .
+
+#Here, is the crux of the previous matter. What does this mean for us and how does it map to NVA? REK and so on? This is covered elsewhere.
+:RegistrationAuthority a rdfs:Class ;
+  rdfs:comment "A standards-setting body from which Registration Agencies obtain official recognition and guidance" ;
+  rdfs:subClassOf :Role .
+
+# Like "contributor"
+:RelatedPerson a rdfs:Class ;
+  rdfs:comment "A contribution made without a specifically defined role in the development of the resource, but who is someone the author wishes to recognize" ;
+  rdfs:subClassOf :Role .
+
+:Researcher a rdfs:Class ;
+  rdfs:comment "A contribution in analyzing data or the results of an experiment or formal study. May indicate an intern or assistant to one of the authors who helped with research but who was not so “key” as to be listed as an author" ;
+  rdfs:subClassOf :Role .
+
+# Again, an organization, not an individual. In my view, this is more a case of something to which a contributor belongs, rather than an actual contributor
+:ResearchGroup a rdfs:Class ;
+  rdfs:comment "Typically refers to a group of individuals with a lab, department, or division; the group has a particular, defined focus of activity" ;
+  rdfs:subClassOf :Role .
+
+# Our understanding of this has been implicitly that this is the Creators of a resource (and their institution?). Our legal team should provide guidance here.
+:RightsHolder a rdfs:Class ;
+  rdfs:comment "Person or institution owning or managing property rights, including intellectual property rights over the resource." ;
+  rdfs:subClassOf :Role .
+
+# Not sure that this doesn't relate to the parent institutions for the researchers, but it may be an individual or an organization
+:Sponsor a rdfs:Class ;
+  rdfs:comment "Person or organisation that issued a contract or under the auspices of which a work has been written, printed, published, developed, etc" ;
+  rdfs:subClassOf :Role .
+
+:Supervisor a rdfs:Class ;
+  rdfs:comment "A contribution by a designated administrator over one or more groups/teams working to produce a resource or over one or more steps of a development process." ;
   rdfs:subClassOf :Role .
 
-:ContributorRole a rdfs:Class ;
-  rdfs:comment "The base class for all contributor roles" ;
+# The implicit constraint here is that the resource is a work package, do we have this resource type, or do we mean something else by this?
+:WorkPackageLeader a rdfs:Class ;
```

Comment:
part of the project datamodell - relevant on the resource when it is exported and released from the surrounding data model